File: #4 - Clean Architecture Version (Bloc)/lib/features/post/domain/entities/post_entity.dart
import 'package:flutter/foundation.dart' show immutable;

@immutable
class PostEntity {
  const PostEntity({
    required this.id,
    required this.userId,
    required this.title,
    required this.body,
  });

  final int id;
  final int userId;
  final String title;
  final String body;
}

File: gorest-backend/src/main/kotlin/com/sinasys/gorest/backend/entity/PostEntity.kt
package com.sinasys.gorest.backend.entity

import jakarta.persistence.Id
import jakarta.persistence.Table
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType


@Entity
@Table(name = "post")
open class PostEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    open var id: Long = 0,
    @Column(nullable = false)
    open var title: String = "",
    @Column(nullable = false, columnDefinition = "TEXT")
    open var body: String = "",
    @Column(nullable = false)
    open var userId: Long? = 10
)